### ISSUE TITLE
After update I get migration error

### ISSUE THREAD

**Issue by janisbarons** [jeremykenedy/laravel-roles#40]
Hi. I recently run composer update on my project. then I run artisan migrate command and got:

`Illuminate\Database\QueryException  : SQLSTATE[HY000]: General error: 1215 Cannot add foreign key constraint (SQL: alter table `role_user` add constraint `role_user_user_id_foreign` foreign key (`user_id`) references `users` (`id`) on delete cascade)`

i looked up the source and see that RolesServiceProvider.php has this line:

`        $this->loadMigrationsFrom(__DIR__.'/Database/Migrations');`

which i think couses the error.

I switched back to previous version of package, but its limiting some features.

**Comment by ekawas** [comment]
Yeah, this is an issue; It seems if you follow the installation steps from the readme, it copies the Migrations from your package to the migrations folder in our project ... Aside from the timestamps being 2016 (which cause them to run first), I see the same exception. Killing the migrations in vendor/jeremykenedy/..../Migrations folder does the trick ...
